device/bgp/bgpv4srtepolicy.yaml

```diff
@@ -0,0 +1,80 @@
+components:
+  schemas:
+    Bgp.V4Policy:
+      x-include:
+      - '../../common/common.yaml#/components/schemas/Named.Object'
+      - '../device.yaml#/components/schemas/Device.Active'
+      description: |
+        Configuration for BGP basic Segment Routing Traffic Engineering(SRTE) 
+        policy settings.
+      type: object
+      required: [ipv4_endpoint]
+      properties:
+        distinguisher:
+          description: >-
+            Identifies the SR policy in the context of <Color, Endpoint> tuple. 
+            It is used by the SR Policy originator to make unique multiple 
+            occurrences of the same SR Policy as well as candidate paths of
+            different SR Policies.
+          type: integer
+          default: 1
+        color:
+          description: >-
+            Policy color is used to match the color of the destination
+            prefixes to steer traffic into the SR Policy.
+          type: integer
+          default: 100
+        ipv4_endpoint:
+          description: >-
+            Specifies a single node or a set of nodes (e.g., an anycast
```

Comment:
Can IPv4 be anycast ? Is it something special here or is this applicable only for IPv6 ?

Reply:
corrected. remote endpoint.

Reply:
There is way to do anycast in IPv4 though it is not explicitly supported in IPv4 spec.